www-client/dooble/patches/dooble-2020.05.05.patchset

```diff
@@ -1,14 +1,13 @@
-From 4193db642c95f836627ad0c36426d20d84aaf29d Mon Sep 17 00:00:00 2001
+From be939a4d30fecb5eaa5a035e45da6e41e18d50b4 Mon Sep 17 00:00:00 2001
 From: Gerasim Troeglazov <3dEyes@gmail.com>
 Date: Sat, 27 May 2017 04:59:44 +1000
 Subject: Fix build for haiku
 
-
-diff --git a/dooble.qt5.pro b/dooble.qt5.pro
-index fc800d3..fd01093 100644
---- a/dooble.qt5.pro
-+++ b/dooble.qt5.pro
-@@ -28,12 +28,12 @@ DEFINES         += DOOBLE_LINKED_WITH_LIBSPOTON \
+diff --git a/boot/home/dooble-build/dooble-2020.05.05-orig/1.x/dooble.qt5.pro b/1.x/dooble.qt5.pro
```

Comment:
This patchset is completely wrong. How in the world did you generate it?

Reply:
Patch is fine and works fine.

Reply:
It has completely incorrect paths and oddly reordered sections, that's not acceptable.

Reply:
Fixed.